recipe/meta.yaml

```diff
@@ -7,16 +7,20 @@ package:
 
 source:
   - url: https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-{{ version }}.tar.xz
-    sha256: 55862232a63459bbf56abebde3085ca9aec211b478e891dacea4d6df8cafe80a
+    sha256: 9ffeab95053f9f6995eb3b3da225e88f21c129cd60da002d3f795db70d6d5974
   - url: https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-{{ version }}.tar.xz
-    sha256: 960b7af4585700db0fdd5b843554e11e2564fed9e061f591fae88a7be6446fa3
+    sha256: 1c596289a0d4207380233eba8c36a932c4d1aceba19932937d9b57c24cef89f3
     folder: plugins_base
   - url: https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-{{ version }}.tar.xz
-    sha256: 3aaeeea7765fbf8801acce4a503a9b05f73f04e8a35352e9d00232cfd555796b
+    sha256: af81154b3a2ef3f4d2feba395f25696feea6fd13ec62c92d3c7a973470710273
     folder: plugins_good
 
 build:
   number: 0
+  # Installing and using plugins in the conda environment isn't working for s390x, due to build options not correctly
+  # translating into the location of a helper library at runtime, for this platform.
+  # Audio/video tools aren't needed on s390x anyway.
+  skip: True # [linux and s390x]
```

Comment:
If we are only skipping `s390x`, can we just say `skip: True. # [s390x]`?

Reply:
the `linux and` is customary, see above